newrelic/example_test.go

```diff
@@ -43,3 +36,48 @@ func ExampleNewExporter() {
 	}
 	global.SetTraceProvider(tp)
 }
+
+func ExampleNewExportPipeline() {
+	// Include environment in resource.
+	r := resource.New(
+		kv.String("environment", "production"),
+		standard.ServiceNameKey.String("My Service"),
+	)
+
+	// Assumes NEW_RELIC_API_KEY contains your New Relic API key. This will
+	// error if it does not.
+	traceProvider, controller, err := newrelic.NewExportPipeline(
+		"My Service",
+		[]trace.ProviderOption{
+			trace.WithConfig(trace.Config{
+				// Conservative sampler.
+				DefaultSampler: trace.ParentSample(trace.NeverSample()),
+				// Reduce span events.
+				MaxEventsPerSpan: 10,
+				Resource:         r,
+			}),
+		},
+		[]push.Option{
+			// Increase push frequency.
+			push.WithPeriod(time.Second),
+			push.WithResource(r),
+		},
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer controller.Stop()
+
+	global.SetTraceProvider(traceProvider)
+	global.SetMeterProvider(controller.Provider())
+}
+
+func ExampleInstallNewPipeline() {
+	// Assumes NEW_RELIC_API_KEY contains your New Relic API key. This will
```

Comment:
Similar here - it would be good to be explicit about this being an env var.

Reply:
Updated.